src/network/internal_types.rs

```diff
@@ -35,6 +37,8 @@ pub struct TearDownNetwork {
 pub struct PortForwardConfigGeneric<Ports, IpAddresses> {
     /// id of container
     pub container_id: String,
+    /// id of the network
+    pub id: String,
```

Comment:
Ah I just remembered the json parser is very strict and fields are required by default so the netavark firewalld reload service would fail to parse the config files created with the old version, not a huge deal but I think just adding `#[serde(default)]` to both fields might solve it. The string would be empty but given that iptables code does not use it that would be fine.

Also for consistency I think `network_id` is a better name

Reply:
Fixed